### ISSUE TITLE
autoHide issue

### ISSUE THREAD

**Issue by filippobottega** [filipliwinski/SimpleConsoleProgress#25]
Hi @filipliwinski,

I have found an issue using `autoHide: true` if the console application is started using `cmd.exe` and `/MIN` option.
Please consider a new console application named `SimpleConsoleProgressTest` with this code in `Program.cs`:

``` csharp
// See https://aka.ms/new-console-template for more information
using SimpleConsoleProgress;

Console.WriteLine("Hello, World!");

var total = 20;
for (int i = 0; i < total; i++)
{
  Thread.Sleep(100);
  ProgressBar.Write(i, total, autoHide: true);
}
Console.WriteLine("Process completed.");
Console.ReadLine();
```

Now start the application using this command:

``` bash
> cmd.exe /C start ""  /MIN "SimpleConsoleProgressTest.exe"
```
The result is:

![image](https://github.com/filipliwinski/SimpleConsoleProgress/assets/2587684/acb36b66-3752-4e01-b9f4-425088bf949a)

Have you any idea why using `/MIN` `autoHide` doesn't work?

Thank you for your support,
Filippo

**Comment by filipliwinski** [comment]
Hi @filippobottega,

this is not an issue with the `autoHide` argument or the `/MIN` option of cmd.exe but it happens for all printouts of the progress bar in external terminals. 

This is because some terminals (like cmd.exe) move to a new line by default when they print a full line of text (with the number of characters equal to the length of the line) for both `Console.Write()` and `Console.WriteLine()`, while others don't.

This issue will be fixed in the next patch release, which I will create next week.

Thank you for reporting this.

**Comment by filippobottega** [comment]
Hi @filipliwinski,

thank you for your reply. Please note that in my tests, when I remove the `/MIN` option, the `ProgressBar.Write` command works as expected using `cmd.exe`.
As for `autoHide`, I agree, it's not relevant.
I'm glad to know that a new patch release is in the works.

Thank you for your support,
Filippo

**Comment by filipliwinski** [comment]
Fixed in [release 1.3.2](https://github.com/filipliwinski/SimpleConsoleProgress/releases/tag/v.1.3.2).
